spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -1110,3 +1110,122 @@ definitions:
       - isc_l2c:
           type: s16
           units: s * 2^-35
+
+ - AlmanacCommonContent:
+    short_desc: Common fields for every almanac message.
+    fields:
+        - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier
+        - toa:
+            type: GPSTime
+            desc: Reference time of almanac
+        - ura:
+            type: double
+            units: m
+            desc: User Range Accuracy
+        - fit_interval:
+            type: u32
+            units: s
+            desc: Curve fit interval
+        - valid:
+            type: u8
+            desc: Status of ephemeris, 1 = valid, 0 = invalid
+        - health_bits:
+            type: u8
+            desc: |
+              Satellite health status.
+              GPS: ICD-GPS-200, chapter 20.3.3.3.1.4
+              SBAS: 0 = valid, non-zero = invalid
+              GLO: 0 = valid, non-zero = invalid
```

Comment:
It's not clear to me, from this description, which bit of `health_bits` maps to each constellation; and could you describe (at least abbreviated) whatever is in "ICD-GPS-200, chapter 20.3.3.3.1.4" instead of just referencing it? Additionally, the `valid` bit above says that 1 is valid and 0 is invalid, which is what I would expect from a binary protocol. Can we be consistent here and say that 1 is valid and 0 is invalid in the health bits as well?